MIGRATION.md

```diff
@@ -0,0 +1,89 @@
+# Migrating to the `cloudant-node-sdk` library
+This document is to assist in migrating from the `nodejs-cloudant` (module: `cloudant`) to the newly supported [`cloudant-node-sdk`](https://github.com/IBM/cloudant-node-sdk) (package: `@ibm-cloud/cloudant`). Compatible with Javascript and TypeScript.
+
+## Initializing the client connection
+There are several ways to create a client connection in `cloudant-node-sdk`:
+1. [Environment variables](https://github.com/IBM/cloudant-node-sdk#authentication-with-environment-variables)
+2. [External configuration file](https://github.com/IBM/cloudant-node-sdk#authentication-with-external-configuration)
+3. [Programmatically](https://github.com/IBM/cloudant-node-sdk#programmatic-authentication)
+
+[See the README](https://github.com/IBM/cloudant-node-sdk#code-examples) for code examples on using environment variables.
+
+## Other differences
+1. For storing credentials in a file, `dotenv` package not needed anymore. Check the new operation [here](https://github.com/IBM/cloudant-node-sdk#authentication-with-external-configuration).
+1. No need to fetch database object in advance to perform operations on Documents. For example, in the case of the document update, this would mean calling first `getDocument` to fetch and after `putDocument` to update.
```

Comment:
Does this apply to just documents?
```suggestion
1. Fetching the database object first before performing operations on documents is not required. For example, in the case of updating a document, this would mean calling first `getDocument` to fetch and after `putDocument` to update.
```

Reply:
Not really, there are a bunch of other things like: replicate, db info, view, find.
I also removed `on documents` from the first sentence.
```suggestion
1. Fetching the database object first before performing operations on it is not required. For example, in the case of updating a document, this would mean calling first `getDocument` to fetch and after `putDocument` to update.
```

Reply:
Changed in c706430